mantis-shaded/build.gradle

```diff
@@ -52,5 +52,5 @@ shadowJar {
     mergeServiceFiles()
 }
 
-jar.deleteAllActions()
+jar.enabled = false
```

Comment:
@rpalcolea what's the effect of this change?

Reply:
`deleteAllActions` was meant to remove all actions in a task, similar to `enabled`

`deleteAllActions` was deprecated and removed in newer versions of Gradle so enabled is the natural replacement here